package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
It guarantees the same dependency tree for each developer/checkout/build.
If we delete it, it will use the current states of each package based on the semantic version of each package. So it can vary for each developer for a different time, because you could have dependencies that say up X or higher, I think.

See: https://stackoverflow.com/questions/44297803/what-is-the-role-of-the-package-lock-json
Or: https://stackoverflow.com/questions/54124033/deleting-package-lock-json-to-resolve-conflicts-quickly

WDYT? Either we keep it and make the install/build stable or we GitIgnore it and build with a fresh set of dependencies based on our settings in `package.json`.

Reply:
But this file is outside of the bbj-vscode package

Reply:
True, looks useless, also because of its missing content. I will remove it.